DaylightWidget/WidgetTextView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  WidgetTextView.swift
+//  DaylightWidgetExtension
+//
+//  Created by Wesley Cheung on 08/03/2021.
+//
+
+import SwiftUI
+
+// View for showing weather message and country
+struct WidgetTextView: View {
+    var entry: Provider.Entry
+    
+    var body: some View {
+        ZStack {
+            Color(entry.daylightController.primaryColor).ignoresSafeArea()
+            
+            VStack(alignment: .center, spacing: 12, content: {
+                Text(entry.daylightController.stringMessage!).multilineTextAlignment(.center).font(Font(Theme.light(size: 15)))
```

Comment:
a) Generally modifiers are done on separate lines such as:

```swift
Text(entry.daylightController.stringMessage!)
  .multilineTextAlignment(.center)
  .font(Font(Theme.light(size: 15)))
```

b) I would SUPER not recommend force unwrapping `stringMessage` here. Remember that `!` basically means crash if this isn't here. One thing you can do is use `entry.daylightController.stringMessage ?? " placeholder"` to give it a default value if the message isn't there.